### PR TITLE
MicroShift - set golangci-lint explicitly

### PR DESCRIPTION
**Pull request by copejon**
Golangci-lint cannot access the default directory to create the .cache subdir because the home dir does not exist.  Specify the cache dir under /tmp to workaround this.

Unblocks openshift/microshift#1605

### CODE REVIEW

File: ci-operator/config/openshift/microshift/openshift-microshift-main.yaml [hunk 1]
```diff
@@ -110,9 +110,18 @@ tag_specification:
...
+    - as: verify
+      commands: make verify
+      env:
+      - default: /tmp/
+        name: GOLANGCI_LINT_CACHE
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
```
Comment (pacevedom) [inline ci-operator/config/openshift/microshift/openshift-microshift-main.yaml:124]: My only nit, is this enough for good scheduling? We can always check after a few executions to adjust.

**Review by pacevedom — APPROVED** [review]
/lgtm
/hold
Holding in case you want to check the resources, I am good to look into it in a different PR.

File: ci-operator/config/openshift/microshift/openshift-microshift-main.yaml (hunk 1 above)
Comment (copejon) [reply ci-operator/config/openshift/microshift/openshift-microshift-main.yaml:124]: It's the same as the global values we set for all tests, so it's probably fine. For whatever reason, steps under the test stanza don't inherit the global request values, according to the linter anyway.

**Pull request merged**